thrift/history.thrift

```diff
@@ -375,6 +375,30 @@ struct GetFailoverInfoResponse {
   20: optional list<i32> pendingShards
 }
 
+struct RatelimitUpdateRequest {
```

Comment:
generated files are not updated. `make all`

Reply:
we only generate public-API proto code in this repo, there's nothing to generate.